_includes/weaviate-embeddings-requirements.mdx

```diff
@@ -0,0 +1,7 @@
+To use Weaviate Embeddings, you need:
+
+- A Weaviate Cloud instance running at least Weaviate version `>=1.27.10`, `>=1.28.3` or `>=1.29.0`.
+- A Weaviate client library that supports Weaviate Embeddings:
+    - Python client version `4.9.5` or higher
+    - JavaScript/TypeScript client version `3.2.5` or higher
+    - Go/Java clients are not yet officially supported; you must pass the `X-Weaviate-Api-Key` and `X-Weaviate-Cluster-Url` headers manually upon instantiation as shown below.
```

Comment:
I suspect this came from the Quickstart snippet? In this instance the code example is not co-located. 

Maybe we could just point to the Quickstart or something.